cypress/integration/4_findings.spec.js

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { PLUGIN_NAME } from '../support/constants';
+import sample_document from '../fixtures/sample_document.json';
+import { createDetector } from '../support/helpers.js';
+
+describe('Findings', () => {
+  const ruleTags = ['low', 'windows'];
+
+  before(() => {
+    // TODO - get cy.createMappings to work successfully so cy.createDetector can be used here instead
+    createDetector();
+  });
+
+  it('displays findings based on recently ingested data', () => {
+    // Confirm detector created
+    cy.url().should(
+      'include',
+      'http://localhost:5601/app/opensearch_security_analytics_dashboards#/detector-details'
```

Comment:
Should it be enough to check for `opensearch_security_analytics_dashboards#/detector-details`?